bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -733,4 +733,4 @@ spec:
     name: cert-manager-ca-injector
   - image: quay.io/jetstack/cert-manager-controller:v1.10.2
```

Comment:
The related images need to updated as well.

Reply:
Updated.